Spawn-App-iOS-SwiftUI/ViewModels/ChoosingTags/ChoosingTagViewModel.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ChoosingTagViewModel.swift
+//  Spawn-App-iOS-SwiftUI
+//
+//  Created by Michael Tham on 23/1/25.
+//
+
+import Foundation
+
+class ChooseTagPopUpViewModel: ObservableObject {
+	@Published var chooseTagErrorMessage: String = ""
+    @Published var tags: [UUID] = []
```

Comment:
```suggestion
    @Published var tags: [FriendTag] = []
```
Let’s use the whole type, to avoid subsequent API calls.